lib/idseq-dag/idseq_dag/steps/generate_coverage_viz.py

```diff
@@ -35,6 +35,7 @@ def run(self):
         max_num_bins_coverage = self.additional_attributes.get("max_num_bins_coverage", MAX_NUM_BINS_COVERAGE)
         num_accessions_per_taxon = self.additional_attributes.get("num_accessions_per_taxon", NUM_ACCESSIONS_PER_TAXON)
         min_contig_size = self.additional_attributes.get("min_contig_size", MIN_CONTIG_SIZE)
+        is_long_read_run = self.additional_attributes.get("is_long_read_run", False)
```

Comment:
It might be more straightforward to name this to directly describe what the flag does. i.e. `keep_taxons_with_only_reads` or something

Reply:
Good point!